funnel/views/account.py

```diff
@@ -91,6 +94,54 @@ def account(self):
     def saved(self):
         return {'saved_projects': current_auth.user.saved_projects}
 
+    @route('siteadmin/comments', endpoint='siteadmin_comments', methods=['GET', 'POST'])
+    @requires_login
+    @render_with('siteadmin_comments.html.jinja2')
+    def siteadmin_comments(self):
+        if not (
+            current_auth.user.is_comment_moderator
+            or current_auth.user.is_user_moderator
+        ):
+            return redirect('/')
+
+        comments = []
+        comment_search_form = CommentSearchForm()
+
+        if comment_search_form.validate_on_submit():
+            comment_search_form.form_nonce.data = (
+                comment_search_form.form_nonce.default()
+            )
+            query = comment_search_form.query.data
+            comments = Comment.query.filter(Comment.search_vector.match(query)).all()
+
+        return {
+            'comments': comments,
+            'comment_search_form': comment_search_form,
+            'default_form': Form(),
+        }
+
+    @route(
+        'siteadmin/comments/delete',
+        endpoint='siteadmin_comments_delete',
+        methods=['GET', 'POST'],
+    )
+    @requires_login
+    @render_with('siteadmin_comments_delete.html.jinja2')
+    def siteadmin_comments_delete(self):
+        if not (
+            current_auth.user.is_comment_moderator
+            or current_auth.user.is_user_moderator
+        ):
+            return redirect('/')
+
+        Form()
```

Comment:
?

Reply:
This was because of flake8 and pre-commit. Temporary change just to make a commit.